resources/js/components/pages/index.vue

```diff
@@ -40,26 +42,49 @@ export default {
   },
   data() {
     return {
-      polls: [
-        {
-          id: -1,
-          name: "Loading polls...",
-          readyUserCount: 0,
-          totalUserCount: "?"
-        }
-      ],
-      closedPolls: [
-        {
-          id: -1,
-          name: "Loading past polls...",
-          readyUserCount: 0,
-          totalUserCount: "?"
-        }
-      ]
+      polls: [],
+      closedPolls: []
     };
+  },
+  mounted() {
+    this.axios
+      .get("/polls")
+      .then(response => {
+        const polls = response.data ? response.data.data : [];
+
+        this.polls = [];
+        this.closedPolls = [];
+
+        for (let i = 0; i < polls.length; i++) {
+          const poll = polls[i];
+          const pollData = {
+            id: poll.id,
+            name: poll.name,
+            readyUserCount: poll.user_count,
+            totalUserCount: poll.total_user_count
+          };
+          if (poll.id === -1) {
```

Comment:
What is this supposed to check ?

Reply:
Fixed, it was supposed to check if an item was choosen